Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,25 @@
+//: Playground - noun: a place where people can play
+
+import SwiftCheck
+import XCTest
+
+//: # Prerequisites
+
+//: This tutorial assumes that you have a fairly good grasp on Swift, its syntax, and terms like 
+//: 
+//: * [(Data) Type](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/TheBasics.html)
+//: * [Protocol-Oriented Programming](https://developer.apple.com/videos/wwdc/2015/?id=408)
+//: * [Test-Driven Development](https://en.wikipedia.org/wiki/Test-driven_development)
+//: * [Polymorphism](https://en.wikipedia.org/wiki/Polymorphism)
+//: * [Map/Filter/Reduce](https://medium.com/@ivicamil/higher-order-functions-in-swift-part-1-d8e75f963d13)
+//:
+//: What this tutorial does *not* require are that you know
+//:
+//: * [Abstract Nonsense](https://en.wikipedia.org/wiki/Abstract_nonsense)
+//: * Functional Programming
+//: * What that dumb 'M' word is/does/means
+
+//: # Introduction
```

Comment:
@rnapier I know you stressed pre-reqs in the last pull request.  Is there anything you would add/remove from this list?  Any better links than wikipedia would be helpful too.